howdy/src/config.ini

```diff
@@ -3,7 +3,7 @@
 
 [core]
 # Print that face detection is being attempted
-detection_notice = false
+detection_notice = true
```

Comment:
I suggest doing this as this message will now only be shown to the user who created the face model. And it looks quite consistent in my case, same as pam_fprintd.so behavior:

https://github.com/boltgolt/howdy/assets/18342621/15618728-9fc1-45bc-a134-53d1578c9f26

Of course, if you insist, I will remove it.

Reply:
Oh, I understand now, that's indeed a good idea for graphical application, but it would also appear on sudo prompt for example, not sure if that's something we want.

Reply:
What do you think of having an argument that we can give to the PAM module configuration that would make the message appear if it's present? This way, we could have it enabled only for certain services.

Reply:
I think there should be either an argument for PAM module or an option in config.ini, but not both at once, otherwise it will confuse the user - for example, he changed the option in config.ini but does not observe any effect.

Reply:
This value is set to false by default because it just crashes some very poorly written lockscreens if it is on. Because of this i think it's better to leave it off if you can agree to that